components/browser/menu/src/test/java/mozilla/components/browser/menu/WebExtensionBrowserMenuBuilderTest.kt

```diff
@@ -284,6 +288,73 @@ class WebExtensionBrowserMenuBuilderTest {
         assertNotNull(addOnsManagerMenuItem)
     }
 
+    @Test
+    fun `extensions sub-menu is moved in place of the placeholder with the corresponding id WebExtensionPlaceholderMenuItem MAIN_EXTENSIONS_MENU_ID `() {
```

Comment:
Let's use [Given/When/Then naming](https://martinfowler.com/bliki/GivenWhenThen.html)

Reply:
Should I also change all test naming in the file?